Update SANS Testing Instructions

DavidFair opened the pull request:
**Description of work.**

    Reword confusing and out of date points from the SANS testing
    instructions

**To test:**
- Read updated docs

<!-- Ensure the base of this PR is correct (e.g. release-next or master)
Finally, don't forget to add the appropriate labels, milestones, etc.!  -->

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

DavidFair committed:
Rename SANSGUITests to ISISSansGUITests

As there is an ORNL and ISIS SANS folder

DavidFair committed:
Clarify ISIS SANS Testing instructions

Reword confusing and out of date points from the SANS testing
instructions

cailafinn reviewed (changes requested):
Just the one minor comment, but otherwise looks much more clear

DavidFair committed:
Remind users to take a note of centre positions

This is because running the beam centre finder will change the values,
and we later ask users to check they now differ

cailafinn reviewed (approved):
Looks good to me 👍
